src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheckTest.java

```diff
@@ -91,17 +93,18 @@ public void testMemberExcludes()
             throws Exception {
         final DefaultConfiguration checkConfig =
             createModuleConfig(AvoidStaticImportCheck.class);
-        checkConfig.addAttribute("excludes", "java.io.File.listRoots");
+        checkConfig.addAttribute("excludes", "java.io.File.listRoots,java.lang.Math.E");
         // allow the java.io.File.listRoots member imports
```

Comment:
this comment is outdated now (if Math.E is added), probably should be removed

Reply:
I've added the `Math.E` also in the comment, as all the tests have comments for property addition